package.json

```diff
@@ -1,28 +1,26 @@
 {
-  "name": "vueposu",
+  "name": "vueposu-monorepo",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "version": "0.3.8-3",
-  "description": "A hooks library based on Vue Composition-API",
-  "repository": "git@github.com:Awesome-Creators/vueposu.git",
-  "homepage": "https://vueposu.netlify.app",
+  "description": "Monorepo by Vueposu",
```

Comment:
Maybe we can remove this description because it doesn't show up anywhere, would you agree?

Reply:
> I think we should reconsider this description. This grammar doesn't feel right . 🧐

Maybe we can remove this description because it doesn't show up anywhere, would you agree?

Reply:
Maybe we can remove this description because it doesn't show up anywhere, would you agree?

Reply:
ok